addon/components/cfb-form-editor/question/validation.js

```diff
@@ -0,0 +1,51 @@
+import Component from "@ember/component";
+import { computed } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { task } from "ember-concurrency";
+import layout from "../../../templates/components/cfb-form-editor/question/validation";
+import allFormatValidatorsQuery from "ember-caluma/gql/queries/all-format-validators";
+
+export default Component.extend({
+  classNames: ["format-validators"],
+  layout,
+
+  apollo: service(),
+
+  async didReceiveAttrs() {
+    this._super(...arguments);
+
+    await this.get("availableFormatValidators").perform();
+  },
```

Comment:
1. I think the `didReceiveAttrs` is the wrong hook for this, since it will perform every time an argument changes but the task has nothing to do with those. I'd prefer using `init`.
2. You don't need to await the task. This will only block rendering which is not very good UX